web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml

```diff
@@ -16,21 +16,19 @@ permissions:
   - 'request collection archival'
   - 'update group'
   - 'view any unpublished content'
-  - 'view any unpublished news content'
   - 'create custom_page content'
   - 'edit any custom_page content'
   - 'delete any custom_page content'
   - 'edit any document content'
   - 'delete any document content'
   - 'edit any event content'
   - 'delete any event content'
-  - 'create rdf entity news'
+  - 'create rdf entity content'
```

Comment:
Why is this changed here? This ticket is about editing your own collections, nothing is mentioned about creating rdf entity content. I'm not even sure what that means, "rdf entity content" sounds very generic. Adding solutions to collections perhaps? This is out of scope for this ticket and there are no tests added for this change.